dataactcore/scripts/pipeline/generate_sam_fabs_export.py

```diff
@@ -0,0 +1,284 @@
+import argparse
+import re
+import logging
+import os
+import csv
+import boto3
+import datetime
+import json
+
+from dataactcore.broker_logging import configure_logging
+from dataactcore.interfaces.db import GlobalDB
+from dataactcore.config import CONFIG_BROKER
+from dataactvalidator.health_check import create_app
+
+logger = logging.getLogger(__name__)
+
+'''
+This script is used to pull updated financial assistance records (from --date to present) for SAM.
+It can also run with --auto to poll the specified S3 bucket (BUCKET_NAME/BUCKET_PREFIX}) for the most
+recent file that was uploaded, and use the boto3 response for --date.
+'''
+
+BUCKET_NAME = CONFIG_BROKER['data_extracts_bucket']
+BUCKET_PREFIX = 'fsrs_award_extracts/'
+
+
+def get_award_updates(mod_date):
+    """ Runs the SQL to extract new award information for SAM
+
+        Args:
+            mod_date: a string in the mm/dd/yyyy format of the date from which to run the SQL
+
+        Returns:
+            The results of the SQL query
+    """
+    logger.info("Starting SQL query of financial assistance records from {} to present...".format(mod_date))
+    sess = GlobalDB.db().session
+    # Query Summary:
```

Comment:
I think this description is still mostly right except that it's not the `latest` transaction but rather all the transactions within the modified_datez